openmetadata-ui/src/main/resources/ui/public/locales/en-US/Database/Hive.md

```diff
@@ -57,24 +74,27 @@ Additional connection arguments such as security or protocol configs that can be
 
 Username to connect to Postgres. This user should have privileges to read all the metadata in Postgres.
 
-### Auth Config
+$$section
+### Auth Config $(id="authType")
 There are 2 types of auth configs:
 - Basic Auth.
 - IAM based Auth.
+- Azure Based Auth.
 
-User can authenticate the Postgres Instance with auth type as `Basic Authentication` i.e. Password **or** by using `IAM based Authentication` to connect to AWS related services.
-
-### Basic Auth
+User can authenticate the Postgres Instance with auth type as `Basic Authentication` i.e. Password **or** by using `IAM based Authentication` to connect to AWS related services **or** by using `Azure Baed Authentication` to connecto to Azure releated services.
+$$
 
+## Basic Auth
 
+### Password $(id="password")
 
-### Password
+Password to connect to Postgres/MySQL.
 
-Password to connect to Postgres.
```

Comment:
section will be required

Reply:
@ayush-shah There are multiple references for Passwords, Username & few other IDs

Reply:
Cool, makes sense. LGTM